partd/pandas.py

```diff
@@ -104,16 +107,21 @@ def serialize(df):
     Uses Pandas blocks, snappy, and blosc to deconstruct an array into bytes
     """
     blocks, index, index_name, columns, placement = to_blocks(df)
+    categories = [b.categories if isinstance(b, pd.Categorical) else None
+                  for b in blocks]
+    blocks = [b.codes if isinstance(b, pd.Categorical) else b
+              for b in blocks]
```

Comment:
Resolved.  FYI I ran into an issue that assert_frame_equal did not pick up on the ordered difference between two Categorical objects.

```python
In [1]: import pandas as pd

In [2]: pd.util.testing.assert_frame_equal(pd.DataFrame({'x': pd.Categorical([], ordered=True)}), pd.DataFrame({'x': pd.Categorical([], ordered=False)}))

In [3]: 
```